lib/build.js

```diff
@@ -94,9 +94,7 @@ const build = (buildConfig = config.defaultBuildConfig, options) => {
   touchOverriddenFiles()
   touchOverriddenVectorIconFiles()
   util.updateBranding()
-  if (buildConfig === 'Release') {
-    licensing.updateLicenses()
-  }
+  licensing.updateLicenses()
```

Comment:
seems like this should go in create_dist with no guard and then it should run for all PRs and that seems like a more appropriate place for it anyway

Reply:
I was aiming to make this fail as early as possible (i.e. anytime a developer builds) so that a simple mistake can't get accidentally get merged to master and break CI builds.

Given the 4 ms overhead, that seems like a reasonable trade-off?